.github/workflows/devpackpublish.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - name: Set all other packages to same version
         env:
           PACKAGE_VERSION: ${{ steps.package_version.outputs.package_version }}
-        run: yarn workspaces foreach -pt --no-private exec npm version --no-git-tag-version $PACKAGE_VERSION
```

Comment:
That would also set the version for the root package, what has been done right before. Not sure if that is a problem, though.

Reply:
The `set:version` does exactly the same as this line that is being replaced, including changing the root package version. Why would this be a problem?